git-branchless/src/commands/move.rs

```diff
@@ -338,7 +322,7 @@ pub fn r#move(
                 }
             }
 
-            builder.move_range(component_root, component_head, component_dest_oid)?;
+            builder.move_subtree(component_root, component_dest_oid)?;
```

Comment:
The children of each component are already being moved (in the `for` loop that immediately precedes this statement), so the call to `move_range` was just unnecessary.